eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/AbstractHTTPServer.java

```diff
@@ -132,13 +142,26 @@ public void sendError(ChannelHandlerContext ctx,
                 "; charset=" + EventMeshConstants.DEFAULT_CHARSET);
         response.headers().add(HttpHeaderNames.CONTENT_LENGTH, response.content().readableBytes());
         response.headers().add(HttpHeaderNames.CONNECTION, HttpHeaderValues.KEEP_ALIVE);
-        // todo server span end with error, record status, we should get channel here to get span in channel's context in async call..
+        //todo server span end with error, record status, we should get channel here to get span in channel's context in async call..
```

Comment:
You may need to add a switch to determine whether you want to enter here

Reply:
fixed